auth/google.go

```diff
@@ -15,49 +15,101 @@
 package auth
 
 import (
+	"encoding/json"
 	"fmt"
+	"log"
 	"net/http"
+	"os"
 	"os/user"
 	"path/filepath"
 
-	"github.com/coreos/mantle/Godeps/_workspace/src/code.google.com/p/goauth2/oauth"
+	"golang.org/x/oauth2"
```

Comment:
You need to run `godep save -r ./...` to rewrite this path and vendor the new oauth2 library under the Godeps directory.